tensorflow/c/tf_tensor.h

```diff
@@ -19,6 +19,7 @@ limitations under the License.
 #include <stdbool.h>
 #include <stdint.h>
 
+#include "tensorflow/c/c_api_macros.h"
```

Comment:
`./tensorflow/c/tf_tensor.h:22:10: error: module //tensorflow/c:c_api_internal does not depend on a module exporting /tensorflow/c/c_api_macros.h'`